src/gg.regression.unity.bots/Runtime/Scripts/RGUtils.cs

```diff
@@ -15,6 +15,14 @@ namespace RegressionGames
 {
     public static class RGUtils
     {
+        // <summary>
+        // Determines if the game is currently running on an Android or Apple mobile device
+        // </summary>
+        public static bool IsMobile()
+        {
+            return Application.platform == RuntimePlatform.Android || Application.platform == RuntimePlatform.IPhonePlayer;
```

Comment:
I don't mind this, but this won't solve the problem for developers making mobile games in the editor. I suppose my biggest concern is more "portrait mode" than mobile specifically (i.e. just making sure our UI is usable for vertical devices)

Reply:
ah I see where you're coming from. I'll make some updates and re-ping you 👍🏻